Task_TheFlexGame/index.css

```diff
@@ -1 +1,85 @@
-/* Start coding below for FLEX GAME TASK*/
+.container{

```

Comment:
Consider using a percentage-based width value instead of a fixed pixel width to ensure better responsiveness.